torchmultimodal/modules/encoders/swin_transformer_3d_encoder.py

```diff
@@ -0,0 +1,518 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Consider this is a big module: can we add tests for some key components here?

Reply:
ok, will try to do it

Reply:
Hi @langong347 , I have added some more test, could you help me review them?